docs/platforms/python/integrations/index.mdx

```diff
@@ -108,6 +110,91 @@ The Sentry SDK uses integrations to hook into the functionality of popular libra
 | <LinkWithPlatformIcon platform="python.tryton"        label="Tryton"          url="/platforms/python/integrations/tryton" />        |                  |
 | <LinkWithPlatformIcon platform="python.wsgi"          label="WSGI"            url="/platforms/python/integrations/wsgi" />          |                  |
 
-## Default Integrations
-
-- [List of Default Integrations](default-integrations/)
+### Default Integrations
+
+- [List of default integrations](default-integrations/)
+
+## Configuration
+
+### Enabling Integrations
+
+Integrations can be added using the `integrations` config option.
+
+Integrations marked as auto enabled in the above table will be turned on automatically with default configuration. If you want to fine-tune a specific integration's settings (for instance, change Flask's default `transaction_style`), add it to your `integrations` list as you would a non-auto-enabling integration and pass in the desired options.
+
+```python
+import sentry_sdk
+from sentry_sdk.integrations.asyncio import AsyncioIntegration
+from sentry_sdk.integrations.flask import FlaskIntegration
+
+sentry_sdk.init(
+    integrations=[
+        # The Flask integration is auto enabling, but we want to change
+        # transaction_style from the default "endpoint" to "url"
+        FlaskIntegration(transaction_style="url"),
+        # The asyncio integration is not enabled automatically
+        # and needs to be added manually.
+        AsyncioIntegration(),
+    ],
+)
+```
+
+
+### Disabling Integrations
+
+There are two types of integrations that are added automatically:
+
+* **Default integrations** like `logging` or `excepthook` are always enabled,
+  regardless of what packages you have installed. They provide essential
+  SDK functionality like error deduplication or event flushing at interpreter
+  shutdown.
+* **Auto-enabling integrations** like `FlaskIntegration` are automatically added
+  if the SDK detects you have the corresponding package (like Flask) installed.
+
+If you want to disable a specific auto enabling integration, first turn off all
```

Comment:
```suggestion
If you want to disable a specific auto-enabling integration, first turn off all
```

Let's pick one of "auto-enabling" or "auto enabling" to settle on and use it consistently everywhere on this page (I strongly prefer "auto-enabling", but consistency is much more important to me than which version we pick).

Reply:
I am gonna stop inline commenting about these discrepancies, since this is probably easier to fix using a find and replace. Please also make sure to update similar variations, e.g. "auto-enable," "auto-enabled," etc

Reply:
Changed to "auto-enab{ed,ing,e}" everywhere